src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.FeatureCollection.cs

```diff
@@ -227,6 +228,12 @@ bool IHttpBodyControlFeature.AllowSynchronousIO
             }
         }
 
+        MinDataRate IHttpMinRequestBodyDataRateFeature.MinDataRate
```

Comment:
Can this be in the Http1Connection class still?

Reply:
Yep.